samples/compression/lz4/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: lz4 sample
+  description: lz4 data compression library
+
+tests:
+  sample.compression.lz4:
+    platform_allow: nrf52840dk_nrf52840
+    platform_exclude: native_posix qemu_x86
+    harness: compression
+    tags: samples lz4
```

Comment:
I am not sure about `samples` tag, it is used only in few samples and I don't think it has any value. I would rather replace it with `compression`.